app/features/instruction-simulation/index.ts

```diff
@@ -0,0 +1,5 @@
+export { SimulatorCard } from './ui/SimulationCard';
+export { SolBalanceChangesCard } from './ui/SolBalanceChangesCard';
```

Comment:
I don't think it is good to export UI component that is not used anywhere except the current feature. Let's keep exports slim as possible.

It is always better to import exact component directly. Same is for hooks. I see that useSimulator is used only inside the feature. Let's not export it then.

In the future it is much better to import hooks like `@features/instruction-simulation/model/useSimulator`.

Could you, please, adjust this entrypoint accordingly?